servers/rendering/shader_language.cpp

```diff
@@ -9061,6 +9064,51 @@ Error ShaderLanguage::_validate_precision(DataType p_type, DataPrecision p_preci
 	return OK;
 }
 
+bool ShaderLanguage::_parse_hint_range_constant(const FunctionInfo &p_constants, float &r_constant) {
+	ShaderLanguage::Node *expr = _parse_and_reduce_expression(nullptr, p_constants);
+	if (expr == nullptr || (expr->type != Node::NODE_TYPE_CONSTANT && expr->type != Node::NODE_TYPE_OPERATOR && expr->type != Node::NODE_TYPE_VARIABLE)) {
+		_set_error(RTR("Expected an integer expression after ','."));
```

Comment:
"Integer" doesn't seem correct if it accepts a general real here, this would be misleading in cases where you provide something that isn't an expression but the value is allowed to be non-integer

Reply:
`Expected a valid numeric expression after ','.` is fine?

Reply:
Should be, what is the error for example after a declaration like `int x = `?

Reply:
Expected expression: found 'SYMBOL`